sites/offshore-mag/server/templates/subscribe/index.marko

```diff
@@ -0,0 +1,58 @@
+import GAM from "../../../config/gam";
+
+$ const { config, site } = out.global;
+
+$ const type = "subscribe";
+$ const title = "Subscribe";
+$ const description = `${config.siteName()} has products that deliver powerful content to you in a variety of forms including print, online, email and social media. To subscribe to a product or manage your current subscription, click below and put the power of ${config.siteName()} at your fingertips.`;
+
+<marko-web-default-page-layout type=type title=title description=description>
+  <@head>
+    <marko-web-gtm-default-context|{ context }| type=type>
+      <marko-web-gtm-push data=context />
+    </marko-web-gtm-default-context>
+  </@head>
+  <@above-container>
+    <marko-web-gam-out-of-page-ad ...GAM.getAdUnit({ name: "wa" }) />
+    <marko-web-gam-out-of-page-ad ...GAM.getAdUnit({ name: "reskin" }) />
+  </@above-container>
+  <@page>
+    <marko-web-gam-define-display-ad ...GAM.getAdUnit({ name: "lb1" }) modifiers=["top-of-page"] />
+    <marko-web-page-wrapper class="mb-block">
+      <@section>
+        <div class="row">
+          <div class="col">
+            <default-theme-breadcrumbs-with-home>
+              <@item>${title}</@item>
+            </default-theme-breadcrumbs-with-home>
+            <h1 class="page-wrapper__title">Information at Your Fingertips</h1>
+            <p class="page-wrapper__description">${description}</p>
+            <default-theme-social-icons
+              items=site.getAsArray('socialMediaLinks')
+              icon-modifiers=["xl"]
+            />
+            <hr>
+          </div>
+        </div>
+      </@section>
+      <@section>
+        <div class="row">
+          <div class="col">
+            <h4>Newsletters</h4>
+            <p>Stay up-to-date on industry news and events, new product launches and more. Select from our list of targeted newsletter offerings to stay informed on the topics that matter to you and your business.</p>
+            <p><strong><marko-web-link href="https://endeavor.dragonforms.com/loading.do?omedasite=LFWPrefPage" target="_blank">Subscribe / Manage Preferences</marko-web-link></strong></p>
```

Comment:
This should be Offshore's subscription page, not LFW

Reply:
updated for all 3